src/aggregator/aggregator/follower_flush_mgr.go

```diff
@@ -267,7 +281,21 @@ func (mgr *followerFlushManager) CanLead() bool {
 			if waitTillFlushedTime.Equal(mgr.openedAt) {
 				waitTillFlushedTime = waitTillFlushedTime.Add(-windowSize)
 			}
-			for _, lastFlushedNanos := range fbr.ByNumForwardedTimes {
+
+			for numForwardedTimes, lastFlushedNanos := range fbr.ByNumForwardedTimes {
+				if lastFlushedNanos == 0 {
+					mgr.logger.Warn("Encountered a window that was never flushed by leader",
```

Comment:
Maybe should just be `Info`? It's not unexpected.

Reply:
fair!